tests/test_resdatafiles.py

```diff
@@ -75,6 +75,15 @@ def test_filedescriptors():
     assert len(list(fd_dir.glob("*"))) == pre_fd_count
     assert resdatafiles._rftfile is None
 
+    deck = resdatafiles.get_deck(sections=[opm.io.eclSectionType.PROPS])
```

Comment:
These new lines should probably be in a new test function, it is not related to testing of file descriptors.